File: Rails_Full_Version/app/controllers/mailbox_controller.rb
class MailboxController < ApplicationController
  def inbox
  end

  def email_view
  end

  def compose_email
  end

  def email_templates
  end

  def basic_action_email
    render :layout => false
  end

  def alert_email
    render :layout => false
  end

  def billing_email
    render :layout => false
  end

end

File: ASPNET_MVC5_Full_Version/Inspinia_MVC5/Controllers/MailboxController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Inspinia_MVC5.Controllers
{
    public class MailboxController : Controller
    {

        public ActionResult Inbox()
        {
            return View();
        }

        public ActionResult EmailView()
        {
            return View();
        }
    
        public ActionResult ComposeEmail()
        {
            return View();
        }
    
        public ActionResult EmailTemplates()
        {
            return View();
        }

        public ActionResult BasicActionEmail()
        {
            return View();
        }

        public ActionResult AlertEmail()
        {
            return View();
        }

        public ActionResult BillingEmail()
        {
            return View();
        }
	}
}